### DISCUSSION TITLE
Moving documentation to a separate repository is better?

### DISCUSSION THREAD

**Discussion by blackcater** [mantinedev/mantine#111] [Q&A] [answered]
Because mantine may have pre-release version now, so documentation also need to have pre-release version.

Then in the future, if we want to support international, a separate repository is more flexible.

What do you think?

**Comment by rtivital** [comment, marked as answer]
From my experience It's much easier to maintain documentation in the same repository as it can be deeply integrated with packages and follows the same release schedule. If we need to deploy documentation for prereleases then it could be done from the branch of that release (in this case documentation will be migrated from gh-pages to Netlify as it supports branches preview) but right now I cannot think why docs should be deployed for prerelease as all releases are kept small and simple (and I think that it stays the same in future). 

Regarding localization – I would prefer not to do it as it brings lots of complexity to documentation management. Let's say I've built new component – now I can write and maintain docs myself, but in case it's translated to another language then this change has to be postponed until the translations are complete. I don't think that project should sacrifice agility to support this.
